controls/entry/text-input.md

```diff
@@ -0,0 +1,34 @@
+---
+title: TextInput
+page_title: .NET MAUI TextInput Documentation - TextInput
+description: Learn more about the TextInput control used inside the Telerik UI for .NET MAUI Entry controls.
+position: 25
+slug: entry-textinput
+---
+
+# TextInput
+
+The TextInput for .NET MAUI is a control which inherits from the Microsoft .NET MAUI `Entry` control. The Telerik UI for .NET MAUI `RadEntry` control uses the TextInput control in its control template for the text input. The `RadEntry` is a templated control, with a `ControlTemplate`, and in that `ControlTemplate` there are borders, close button, and a TextInput.
+
+The TextInput control exposes the following properties:
+
+* `SelectionOnFocus` (`enum` of type `Telerik.Maui.SelectionOnFocus`)&mdash;Specifies the text selection behavior when the Entry control receives focus. The available options are:
+    * (Default) `Unchanged`&mdash;the selection will not be modified.
```

Comment:
Use a capital letter after the em dash. This comment applies to all the items that follow.
```suggestion
    * (Default) `Unchanged`&mdash;The selection will not be modified.
```

Reply:
The comment applies to all items in the list.